ra/registration-authority.go

```diff
@@ -787,9 +789,39 @@ func (ra *RegistrationAuthorityImpl) UpdateAuthorization(base core.Authorization
 	}
 
 	// Dispatch to the VA for service
-	ra.VA.UpdateValidations(authz, challengeIndex)
 
-	ra.stats.Inc("RA.UpdatedPendingAuthorizations", 1, 1.0)
+	if !ra.useNewVARPC {
+		ra.VA.UpdateValidations(authz, challengeIndex)
+		ra.stats.Inc("RA.UpdatedPendingAuthorizations", 1, 1.0)
+	} else {
+		go func() {
+			records, err := ra.VA.PerformValidation(authz.Identifier.Value, authz.Challenges[challengeIndex], authz)
+			var prob *probs.ProblemDetails
+			if p, ok := err.(*probs.ProblemDetails); ok {
+				prob = p
+			} else if err != nil {
+				prob = probs.ServerInternal("uncaught error in RPC to VA")
```

Comment:
Rather than uncaught error, which is rather alarming, let's just call this "Problem communicating with VA." Also, local style has been to use initial caps for problem details.